templates/easy_admin/Conference/country.html.twig

```diff
@@ -1 +1,5 @@
-<img class="country-flag" alt="{{ country_name }} flag" title="{{ country_name }}" src="/bundles/easyadmin/images/flags/{{ value }}.png">
+{% if value|lower == 'online' %}
```

Comment:
🤔 I don't like that because this logic will be spread in all the application.
Please add a method in the class to do that. 
And may be it should be a flag in the class and not a hack with the city / country... Choose the best options here